src/server/controllers/continue.controller.js

```diff
@@ -103,8 +104,12 @@ const continueController = (
       pathFromSession
     );
 
-    // update the new path to switch off manual address input pages if the originator (currentPage) is one of the address select pages
-    const updatedNewPath = switchOffManualAddressInput(newPath, currentPage);
+    // update the new path to switch off manual address input pages if the originator (currentPage) is one of the address select pagees
+    // and switch off representative registration role path when changed to Sole trader
+    const updatedNewPath = switchOffCompanyAndCharityDetails(
+      trimmedNewAnswers,
```

Comment:
As discussed, very minor point: break out into two seperate lines so no implicit dependency and allow for further such operations to be displayed neatly. e.g.
  var updatedNewPath = switchOffManualAddressInput(newPath, currentPage);
  updatedNewPath = switchOffCompanyAndCharityDetails(trimmedNewAnswers, updatedNewPath );